chain/dev/config.toml

```diff
@@ -15,10 +15,10 @@ sync = ""
 digest = ""
 
 [init]
-genesis = "./chain/dev/dev-spec-raw.json"
+genesis = "./chain/dev/genesis-spec.json"
 
 [account]
-key = "alice"
+key = ""
```

Comment:
Why is the key removed?

Reply:
see [comment](https://github.com/ChainSafe/gossamer/pull/2560#discussion_r875106720)